Classes/URKArchive.mm

```diff
@@ -134,7 +134,9 @@ - (instancetype)initWithFile:(NSURL *)fileURL password:(NSString*)password error
         if (error) {
             *error = nil;
         }
-
+        
+        fileURL = [self firstVolumeURL:fileURL];
```

Comment:
Just added it before `(!fileURL)` check and `[self init]`, let me know if you think there's a better place